### ISSUE TITLE
logging into test openshift VM is refused: x509: certificate signed by unknown authority

### ISSUE THREAD

**Issue by martinpitt** [cockpit-project/cockpit#5968]
I was going to get back to some openshift integration bugs, but it seems with some image refresh or cockpit code rework the instructions in `pkg/kubernetes/README.md` stopped working:
```
$ test/vm-run openshift  # works fine
$ cp ./test/verify/files/openshift.kubeconfig ~/.kube/config
$ kubectl get pods
Unable to connect to the server: x509: certificate signed by unknown authority
```
Similarly, the web UI at http://localhost:9090/kubernetes says "Couldn't connect to server: unknown-hostkey".

**Comment by martinpitt** [comment]
Logging in as admin:foobar or any other user/password on https://10.111.112.101:8443 still works fine (once you accept the self-signed SSL cert). However, that just creates a new user role, similarly to creating a new user on http://localhost:9090/kubernetes → Troubleshoot (it doesn't have any privileges then).

**Comment by martinpitt** [comment]
So for sure there is some certificate mismatch problem:
```
$ kubectl --insecure-skip-tls-verify=true get pods
Error from server (Forbidden): User "scruffy" cannot get pods in project "default"
```
The config is supposed to include the authority data which ought to work even for the self-signed cert. But beyond that it seems that the "scruffy" user lost its admin privs somehow?

**Comment by martinpitt** [comment]
Fix in #5970
